src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -516,12 +516,13 @@ private InstructorFeedbackResultsGroupByParticipantPanel buildGroupByParticipant
             if (!viewType.isPrimaryGroupingOfGiverType()) {
                 boolean isStudent = bundle.roster.getStudentForEmail(secondaryParticipantIdentifier) != null;
                 boolean isVisibleTeam = isTeamVisible(secondaryParticipantDisplayableName);
+                boolean isInstructor = bundle.roster.getInstructorForEmail(secondaryParticipantIdentifier) != null;
                 String sectionName = bundle.getSectionFromRoster(secondaryParticipantIdentifier);
-                boolean isAllowedToModerate = (isStudent || isVisibleTeam) 
+                boolean isAllowedToModerate = (isStudent || isVisibleTeam || isInstructor) 
```

Comment:
What is the logic for who is allowed to moderate? Extract the condition out into a function?
Unless i'm mistaken, a person in the course is either student or instructor .. is there a point to isStudent || isInstructor when it will always return true?